variables.tf

```diff
@@ -257,6 +269,12 @@ variable "metadata_cleanup_dry_run_enabled" {
   type        = string
 }
 
+variable "api_dry_run_enabled" {
```

Comment:
Is this a boolean? You should use type boolean, not use a string to compare true or false

Reply:
There is the type `bool`, but all the other boolean variables in these files are specified as a `string` or a `number` which is 0 or 1 so I'm a bit hesitant to change it just for this one?